src/Bridge.ts

```diff
@@ -289,56 +278,48 @@ export class Bridge extends vscode.Disposable {
 
     return true
   }
-  private async installBinaryViaCargo(version: string): Promise<boolean> {
-    this.button.setState(ButtonState.Installing)
-
-    const compileProcess = childProcess.spawn(getCargoPath(), [
-      "install",
-      "--git",
-      ROJO_GIT_URL,
-      "--tag",
-      version,
-      "--root",
-      this.rojoInstallPath
-    ])
-
-    compileProcess.stdout.on("data", data => sendToOutput(data))
-    compileProcess.stderr.on("data", data => sendToOutput(data))
-    outputChannel.show()
 
-    try {
-      await new Promise((resolve, reject) => {
-        compileProcess.on("exit", resolve)
-        compileProcess.on("error", reject)
-      })
-    } catch (e) {
-      console.log(e)
+  private getBinaryZipPattern(): RegExp | undefined {
+    switch (os.platform()) {
+      case "win32":
+        return /^rojo(?:-|-.*-)win64.zip$/
+      case "darwin":
+        return /^rojo(?:-|-.*-)macos.zip$/
+      // case "linux":
+      //   return /^rojo(?:-|-.*-)linux.zip$/
```

Comment:
Linux install is possible from release assets, but I don't think the extension currently supports it (re: the Studio Plugin), so I left this commented out.

Not sure if you want Linux support for installation, or if this should just be removed completely

Reply:
I think supporting linux is fine